app/templates/components/modals/user-delete-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="header">
+  {{t 'Are you sure you would like to delete your account?'}}
+  <div class="muted small text">
+    {{t 'Deleting the account will delete all the data associated with it.'}}
+  </div>
+</div>
+<div class="content">
+  <div class="ui {{if isLoading 'loading'}} form" autocomplete="off">
+    <div class="field">
+      <div class="label">
+        {{t 'Please enter your Email ID to confirm that you want to delete the account'}}
```

Comment:
Move the checkbox in the starting or in the right margin , Checkbox in middle of sentence is depreciating UI 
![image](https://user-images.githubusercontent.com/44091822/57867830-72fccc00-781f-11e9-9dea-94ed3dc813a0.png)

Reply:
@kushthedude I agree. I've made the required changes